### DISCUSSION TITLE
AutoTimer Interval

### DISCUSSION THREAD

**Discussion by caco3** [jomjol/AI-on-the-edge-device#1197] [Q&A]
In the config, it states that the `AutoTimer Interval` must be greater than 3 minutes. 

@jomjol Do you remember why we have this limit?
To me it does not make sense. The only need I see is that it must be longer than the time we need for a full run.
But this really depends on the number of digits to be digitalized. I have set it to 2 minutes on my meters and never saw an issue with this (a run completes within 50s on my meters).

I would suggest to remove this restriction. The `task_autodoFlow` simply should wait for the next interval to expire. And if the interval is set too short, it should start the next run (maybe with some idle time beween if needed).
As far as I can see, there is actually already some code for this in `server_tflite.cpp`!

And also the default value is quite an odd number!
Why not use a clean number like 5?

**Comment by jomjol** [comment]
Basically, we can remove this recommendation. This comes from the time of a rather instable run (restart, ...).
One would need to check, if the code in `server_tflite.cpp` is working in case of intervalls shorter than the cycle time.

**Reply by caco3** [reply]
ok, thanks!

I will test and remove the restriction in the note.
what about the default value?

**Comment by caco3** [comment]
https://github.com/jomjol/AI-on-the-edge-device/issues/1198

**Comment by jalla2000** [comment]
Can the interval be set to 0, and it will update as fast as it can?

**Reply by caco3** [reply]
Not sure if 0 works, but you might be able to set it to eg. 0.1. It then will run one round after the other, see https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-interval
